### DISCUSSION TITLE
How to access extended colors on extended boxShadow

### DISCUSSION THREAD

**Discussion by OrhanOzkercin** [tailwindlabs/tailwindcss#7250] [Help] [answered]
I extended colors object on tailwind.config.js.

```
 extend: {
      colors: {
        primary: "#22d3ee",
        "text-color": colors.slate[400],
      },
```

I also have extended boxShadow field and I want to access primary color on this field like this

```
 boxShadow: {
        "primary-button": `inset 0 0 0 2px ${colors.primary}`,
      },

```

This is not working 
![Code_3vg15DN9qr](https://user-images.githubusercontent.com/25130814/151606583-e2975b31-bcfd-438f-a9c7-9fffad16e30c.png)

**Comment by adamwathan** [comment, marked as answer]
Docs here should help:

https://tailwindcss.com/docs/theme#referencing-other-values

**Reply by OrhanOzkercin** [reply]
Great! Thank you very much 🤙
